bedrock/base/templates/includes/protocol/navigation/menus/products.html

```diff
@@ -12,17 +12,12 @@
     <div class="c-menu-panel-container">
       <button class="c-menu-button-close" type="button" aria-controls="c-menu-panel-products">{{ ftl('navigation-v2-close-products-menu') }}</button>
       <div class="c-menu-panel-content">
-        <ul class="mzp-l-rows-four">
+        <ul class="mzp-l-rows-three">
           <li>
             <section class="c-menu-item mzp-has-icon">
-              {% set mozilla_monitor = switch('mozilla-monitor-brand-name') %}
-              <a class="c-menu-item-link" href="https://monitor.mozilla.org/?{{ utm_params }}" data-link-text="{% if mozilla_monitor %}Mozilla Monitor{% else %}Firefox Monitor{% endif %}" data-link-type="nav" data-link-position="topnav" data-link-group="products">
+              <a class="c-menu-item-link" href="https://monitor.mozilla.org/?{{ utm_params }}" data-link-text="Mozilla Monitor" data-link-type="nav" data-link-position="topnav" data-link-group="products">
                 <img loading="lazy" src="{{ static('protocol/img/logos/firefox/monitor/logo.svg') }}" class="c-menu-item-icon" width="32" height="32" alt="">
-                {% if mozilla_monitor %}
-                  <h4 class="c-menu-item-title">{{ ftl('navigation-v2-mozilla-monitor', fallback='navigation-v2-firefox-monitor') }}</h4>
-                {% else %}
-                  <h4 class="c-menu-item-title">{{ ftl('navigation-v2-firefox-monitor', fallback='navigation-firefox-monitor') }}</h4>
```

Comment:
**Suggestion:** Now that we're using the new `Mozilla Monitor` string here, feel free to remove the (now obsolete) `navigation-v2-firefox-monitor` string from the `navigation_v2.ftl` file :) since it's been (much) more than 2 months (referencing the [new rule we implemented](https://bedrock.readthedocs.io/en/latest/l10n.html#obsolete-strings) of us deciding to remove obsolete strings from our codebase after ~2 months) from when we implemented the new string!

Reply:
👍 Makes sense. I felt the nav is rather "sensitive" to do any cleanups but after wandering around the locales I see the typical pattern: whenever the default string is used, it's amid of 80% untranslated elements in English anyways. And for a branding change, it's way better to have a fresh default, than an outdated name, even if localised. You got it.